api/src/main/java/com/example/api/service/activity/result/TaskResultService.java

```diff
@@ -156,27 +167,30 @@ public ActivityStatisticsResponse getActivityStatistics(Long activityID) throws
         return getActivityStatisticsForActivity(activity);
     }
 
-    private void fillFirstRowAndAddTasksToMap(List<ActivityTypeWithIdForm> forms,
-                                              Map<Long, GraphTask> formToGraphTaskMap,
-                                              Map<Long, FileTask> formToFileTaskMap,
-                                              Map<Long, Survey> formToSurveyMap,
-                                              List<String> firstRow) {
-        forms.forEach(form -> {
-            switch (form.getType()){
-                case EXPEDITION -> {
-                    GraphTask graphTask = graphTaskRepo.findGraphTaskById(form.getId());
-                    firstRow.addAll(List.of("Zadanie:" + graphTask.getTitle() + " (Punkty)", "Zadanie:" + graphTask.getTitle() + " (Informacja zwrotna)"));
-                    formToGraphTaskMap.put(graphTask.getId(), graphTask);
-                }
-                case TASK -> {
-                    FileTask fileTask = fileTaskRepo.findFileTaskById(form.getId());
-                    firstRow.addAll(List.of("Zadanie:" + fileTask.getTitle() + " (Punkty)", "Zadanie:" + fileTask.getTitle() + " (Informacja zwrotna)"));
+    private void fillFirstRowAndAddTasksToMap(List<Long> activityIds,
+            Map<Long, GraphTask> formToGraphTaskMap,
+            Map<Long, FileTask> formToFileTaskMap,
+            Map<Long, Survey> formToSurveyMap,
+            List<String> firstRow) {
+        activityIds.forEach(activityId -> {
+            GraphTask graphTask = graphTaskRepo.findGraphTaskById(activityId);
+            if (graphTask != null) {
```

Comment:
tam wyzej zmienles a tu nie xdd ale chuj z tym juz jak dziala to git

Reply:
Tutaj też teraz zmieniłem, eksport dalej wygląda na działający